.changeset/healthy-ties-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": minor
+---
+
+feat(Input): add onSubmit prop on BaseInput, TextInput, TextArea, OTPInput & PasswordInput for react-native
```

Comment:
I don't see any change on OTPInput files? 🤔

Reply:
I'll remove it